src/cook-web/src/app/c/[[...id]]/Components/ChatUi/AskBlock.module.scss

```
@@ -46,6 +46,11 @@
     font-weight: var(--font-weight-normal, 400);
     line-height: 24px; /* 150% */
   }
+  .askIframeWrapper {
+    :global(.content-render-iframe-sandbox) {
+      height: auto !important;
+    }
+  }
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)